apps/client/src/features/overview/composite/TimeElements.tsx

```diff
@@ -36,7 +45,7 @@ export function StartTimes() {
       <div className={style.row}>
         <span className={style.label}>Start</span>
         <div className={style.labelledElement}>
-          <Tooltip text='Planned start time' render={<TbCalendar className={style.icon} />} />
+          <Tooltip text='Planned start time' render={<TbCalendarPin className={style.icon} />} />
```

Comment:
question: if we are changing the fixed times to have the pin icon, should the end time not have one as well?